squad/frontend/templates/squad/_test_run_test.html

```diff
@@ -1,18 +1,44 @@
+{% load squad %}
 <div class='row row-bordered {{test.status}} tests-{{status.id}}-{{test.status}}' style='display: {% if test.status == "fail" %}block{% else %}none{% endif %}'>
-    <div class='col-md-6 col-sm-6'>
+    <div class='col-md-10 col-sm-10'>
         <i class='fa fa-{% if test.status == "fail" %}times{% elif test.status == "pass" %}check{% else %}question-circle-o{% endif %}'></i>
         <a href="{% url 'test_history' project.group.slug project.slug test.full_name %}">{{test.name}}</a>
     </div>
-    <div class='col-md-6 col-sm-6'>
-        {% if test.log %}
-            <a class="btn btn-sm btn-danger testlog-{{test.id}}" href='#' onclick='$(".testlog-{{test.id}}").toggle(); return false'>Show log snippet</a>
-            <a class="btn btn-sm btn-danger testlog-{{test.id}}" href='#' onclick='$(".testlog-{{test.id}}").toggle(); return false' style='display: none'>Hide log snippet</a>
-        {% else %}
-            <em>No log available for this test</em>
+    <div class='col-md-2 col-sm-2'>
+        {% if test.metadata.description or suite.metadata.instructions_to_reproduce or test.metadata.instructions_to_reproduce or test.log %}
+        <a href='#'
+           onclick='$(".test-description-{{test.id}}").toggle(); return false'
+           class='test-description-{{test.id}}'
+        ><span class='fa fa-info-circle'></span> Show info</a>
+        <a href='#'
+           onclick='$(".test-description-{{test.id}}").toggle(); return false'
+           class='test-description-{{test.id}}'
+           style='display: none'
+        ><span class='fa fa-info-circle'></span> Hide info</a>
         {% endif %}
     </div>
-    <div class='testlog-{{test.id}} col-md-12 col-sm-12' style='display: none'>
+    {% if test.metadata.description or suite.metadata.instructions_to_reproduce or test.metadata.instructions_to_reproduce or test.log%}
+    <div class='test-description test-description-{{test.id}} col-md-12 col-sm-12' style='display: none'>
+        {% if test.metadata.description %}
+        <p><strong>Test description:</strong></p>
+        <div>
+        {{test.metadata.description|markdown}}
+        </div>
+        {% endif %}
+
+        {% if suite.metadata.instructions_to_reproduce or test.metadata.instructions_to_reproduce %}
+        <p><strong>How to reproduce:</strong></p>
+        <div>
+        {{suite.metadata.instructions_to_reproduce|markdown}}
```

Comment:
I can't get the suite.metadata.instructions_to_reproduce displayed. The objects is set properly in the database but doesn't show up in the UI.

Reply:
yes, my mistake. I just pushed a fixed version.